uniffi_docs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "uniffi_docs"
+version = "0.23.0"
+edition = "2021"
+description = "uniffi_meta"
+homepage = "https://mozilla.github.io/uniffi-rs"
+repository = "https://github.com/eigerco/uniffi-rs"
+license = "MPL-2.0"
+keywords = ["ffi", "bindgen", "docs"]
+
+[dependencies]
+anyhow = "1"
+syn = { version = "1.0", features = ["full"] }
+pulldown-cmark = { version = "0.9.2"}
+
+[dev-dependencies]
+indoc = "2"
+quote = "1.0.23"
```

Comment:
Does this need to be a separate crate?  It seems like it could be a submodule of `uniffi_bindgen`, maybe with a feature flag to enable/disable it.

Reply:
Actually, never mind about this one, a separate crate seems good since it allows both `uniffi_bindgen` and `uniffi_macros` to depend on it.